build-logic/binary-compatibility/src/test/kotlin/gradlebuild/binarycompatibility/AbstractAcceptedApiChangesMaintenanceTaskIntegrationTest.kt

```diff
@@ -45,13 +46,13 @@ abstract class AbstractAcceptedApiChangesMaintenanceTaskIntegrationTest {
                     val verifyAcceptedApiChangesOrdering = tasks.register<gradlebuild.binarycompatibility.AlphabeticalAcceptedApiChangesTask>("verifyAcceptedApiChangesOrdering") {
                         group = "verification"
                         description = "Ensures the accepted api changes file is kept alphabetically ordered to make merging changes to it easier"
-                        apiChangesFile.set(layout.projectDirectory.file("${acceptedApiChangesFile.absolutePath}"))
+                        apiChangesFile.set(layout.projectDirectory.file("${ FilenameUtils.normalize(acceptedApiChangesFile.absolutePath, true) }"))
```

Comment:
alternatively, just hardcode the path here again
```suggestion
                        apiChangesFile.set(layout.projectDirectory.file("src/changes/accepted-public-api-changes.json"))
```

Reply:
I think it's worth not hardcoding these if we don't have to.